apps/frontend/src/renderer/components/forgejo-issues/components/EmptyStates.tsx

```
@@ -0,0 +1,53 @@
+import { useTranslation } from 'react-i18next';
+import { Settings2 } from 'lucide-react';
+import { Button } from '../../ui/button';
+import type { EmptyStateProps, NotConnectedStateProps } from '../types';
+
+// Forgejo icon component
+function ForgejoIcon({ className }: { className?: string }) {
+  return (
+    <svg className={className} viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2.5" strokeLinecap="round">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)